x-pack/test/functional/apps/monitoring/_get_lifecycle_methods.js

```diff
@@ -35,7 +36,7 @@ export const getLifecycleMethods = (getService, getPageObjects) => {
 
     async tearDown() {
       await PageObjects.security.forceLogout();
```

Comment:
may be we can remove the forceLogout(), we are just changing the role of one user who has logged in ? wdyt ?